common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -666,8 +670,6 @@
   "feature.wynntils.shamanTotemTracking.secondTotemColor.name": "Shaman Totem 2 Color",
   "feature.wynntils.shamanTotemTracking.thirdTotemColor.description": "What color should the totem 3 highlight and tracking text be?",
   "feature.wynntils.shamanTotemTracking.thirdTotemColor.name": "Shaman Totem 3 Color",
-  "feature.wynntils.shamanTotemTracking.trackTotem.description": "Should Shaman/Skyseer totems be tracked?",
```

Comment:
Should this really be removed?

Reply:
Yeah, it was originally intended for the enable/disable config, but since Artemis just does this by disabling/enabling the feature, we don't need a specific config